admin-node-setup.sh

```diff
@@ -61,6 +61,12 @@ fi
 # Generate TLS CA and Initial Certificates
 /usr/share/caasp-container-manifests/gen-certs.sh
 
+# add an entry for ldap.infra.caasp.local to /etc/hosts
+# this is needed to enable net-ldap to validate the certificate for LDAP_HOST
+if ! [ "$(cat /etc/hosts | grep -E "^127.0.0.1\s+" | grep ldap.infra.caasp.local)" ]; then
+    sed -i 's/127.0.0.1\tlocalhost/127.0.0.1\tlocalhost ldap.infra.caasp.local/g' /etc/hosts
```

Comment:
@ereslibre do we also need to adapt `/etc/hosts` in salt? i am not fully sure if it is still relevant *after* creating the ldap user, but the file will likely be overwritten by salt

Reply:
Yes, it's relevant because you want to login to Velum afterwards. Velum needs to reach the LDAP instance. Before it was `127.0.0.1`, but with this change, `ldap.infra.caasp.local` needs to be "resolvable" at all times, otherwise the login from Velum will fail.

Reply:
Note that I'm talking about the very first login page (not the kubeconfig download that uses dex and that's fine).

Reply:
added in https://github.com/kubic-project/salt/pull/396